includes/admin/class-sensei-extensions.php

```diff
@@ -168,15 +168,29 @@ private function add_installed_extensions_properties( $extensions ) {
 			$wccom_subscriptions = WC_Helper::get_subscriptions();
 		}
 
+		$aliases = [
+			'sensei-pro/sensei-pro.php' => [
+				'woothemes-sensei/woothemes-sensei.php',
+			],
+		];
+
 		// Includes installed version, whether it has update and WC.com metadata.
 		$extensions = array_map(
-			function( $extension ) use ( $installed_plugins, $wccom_subscriptions ) {
-				$extension->is_installed = isset( $installed_plugins[ $extension->plugin_file ] );
-				$extension->is_activated = $extension->is_installed && is_plugin_active( $extension->plugin_file );
+			function( $extension ) use ( $installed_plugins, $wccom_subscriptions, $aliases ) {
+				$plugin_file = $extension->plugin_file;
+
+				$plugin_file_aliases   = array_key_exists( $plugin_file, $aliases ) ? $aliases[ $plugin_file ] : [];
+				$plugin_file_aliases[] = $plugin_file;
 
-				if ( $extension->is_installed ) {
-					$extension->installed_version = $installed_plugins[ $extension->plugin_file ]['Version'];
-					$extension->has_update        = isset( $extension->version ) && version_compare( $extension->version, $extension->installed_version, '>' );
+				foreach ( $plugin_file_aliases as $extension_plugin_file ) {
+					$extension->is_installed = isset( $installed_plugins[ $extension_plugin_file ] );
+					$extension->is_activated = $extension->is_installed && is_plugin_active( $extension_plugin_file );
+
+					if ( $extension->is_installed ) {
+						$extension->installed_version = $installed_plugins[ $extension_plugin_file ]['Version'];
+						$extension->has_update        = $extension_plugin_file === $plugin_file && isset( $extension->version ) && version_compare( $extension->version, $extension->installed_version, '>' );
```

Comment:
I disabled the check for `has_update` when `$extension_plugin_file` refers to an alias (in other words, it doesn't refer to `$plugin_file`) because the versioning scheme used in `woothemes-sensei` is different than the one used in the Sensei Pro, so the comparison here could end up being weird (and probably faulty, too).